setup.py

```diff
@@ -18,7 +17,7 @@ def get_version(*file_paths):
     Extract the version string from the file at the given relative path fragments.
     """
     filename = os.path.join(os.path.dirname(__file__), *file_paths)
-    version_file = open(filename).read()
+    version_file = open(filename).read()  # pylint: disable=open-builtin
```

Comment:
my PR addresses these, as you've seen and commented on. Could you roll them back so I'm not going on PTO with merge conflicts on my PRs?

Reply:
Done.